tasks/abs.rb

```diff
@@ -120,13 +126,14 @@ def tear_down(node_name, inventory_location)
 
 params = JSON.parse(STDIN.read)
 platform = params['platform']
+platforms = params['platforms']
```

Comment:
I'd prefer doing the platform/s coalescing here to keep the business logic in `provision` cleaner:
```suggestion
platform = platforms = params['platforms'] || {params['platform']
 => 1}
```